projects/igniteui-angular/src/lib/expansion-panel/expansion-panel.component.ts

```diff
@@ -229,7 +229,7 @@ export class IgxExpansionPanelComponent implements IgxExpansionPanelBase, AfterC
         }
         this.playCloseAnimation(
             () => {
-                this.onCollapsed.emit({ event: evt, panel: this });
+                this.onCollapsed.emit({ event: evt, panel: this, owner: this });
```

Comment:
please update the comments for this event, as  it is not pointing in TS part the correct type of the event - `IExpansionPanelEventArgs`. It looks like this now:
```
     * ```typescript
     *  handleCollapsed(event: {
     *  panel: IgxExpansionPanelComponent,
     *  event: Event
     * })
     * ```
```